starter/script.js

```diff
@@ -1 +1,176 @@
-// put code here!
+// one hipster's tale
+
+// functions:
+function getAdventure(adventureState) {
+//   input: adventureState, player info
+//   effect: display current adventure
+  console.log(adventureText[adventureState]);
+};
+
+function getResponse(adventureState) {
+//  input: question/response key, playerinfo
+//  returns: option letter
+  getAdventure(adventureState);
+  var a = '';
```

Comment:
Using a more semantic variable name like `answer` or `response` would help the readability, particularly for things like `a != 'a'`